migaloo/assetlist.json

```diff
@@ -790,6 +790,52 @@
           }
         }
       ]
-    }
+    },
+    {
+      "denom_units": [
+        {
+          "denom": "ibc/33E97105B5C1F48CB7D6400F4673F9B3699AC59EDB621CCCD57D5D480E0C83B6",
+          "exponent": 0,
+          "aliases": [
+            "factory/inj1a6xdezq7a94qwamec6n6cnup02nvewvjtz6h6e/SYN"
+          ]
+        },
+        {
+          "denom": "SYN",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/33E97105B5C1F48CB7D6400F4673F9B3699AC59EDB621CCCD57D5D480E0C83B6",
+      "name": "SYN",
+      "display": "SYN",
+      "symbol": "SYN",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "injective",
+            "base_denom": "factory/inj1a6xdezq7a94qwamec6n6cnup02nvewvjtz6h6e/SYN",
+            "channel_id": "channel-102"
+          },
+          "chain": {
+            "channel_id": "channel-3",
+            "path": "transfer/channel-3/factory/inj1a6xdezq7a94qwamec6n6cnup02nvewvjtz6h6e/SYN"
+          }
+        }
+      ],
+      "images": [
+        {
+          "image_sync": {
+            "chain_name": "injective",
+            "base_denom": "factory/inj1a6xdezq7a94qwamec6n6cnup02nvewvjtz6h6e/SYN"
+          },
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/injective/images/syn.ash.png"
```

Comment:
It looks like this asset is SYN, but the image you're pointing to here is syn.ash, which seems incorrect to me. Should just just syn.png?